bundles/worlds/worlds_service.go

```diff
@@ -694,10 +694,18 @@ func parseModelIncludes(tx *gorm.DB, world *World,
 	if err != nil {
 		return nil, gz.NewErrorMessageWithBase(gz.ErrorFormInvalidValue, err)
 	}
-	defer xmlFile.Close()
-	b, _ := ioutil.ReadAll(xmlFile)
+	defer func(file *os.File) {
+		err := file.Close()
+		if err != nil {
+			log.Println("Failed to close file:", err)
+		}
+	}(xmlFile)
```

Comment:
This is something that came to my mind while refactoring this, and I'm glad you raise this comment here. I agree we should extract this to `gz-go`, but I can't avoid asking myself: what happens if you wanna introduce a custom error handling action when failing to close a file?

Reply:
We can have `Close(closer io.Closer)` and `CloseWithHandler(closer io.Closer, errorHandler func())`.

Depending on the complexity of your handler, it might just make more sense to not use the `gz-go` function and write the special case as an exception.